gm4/plugins/annotations.py

```diff
@@ -32,21 +49,81 @@ def filter(record: logging.LogRecord):
 class AnnotationFormatter(logging.Formatter):
     
     def format(self, record: logging.LogRecord) -> str:
+
         expl = record.getMessage().replace("\n", "%0A")
             # use urlencoded newline
         
-        level = LEVEL_CONVERSION.get(record.levelno, logging.INFO)
+        level = LEVEL_CONVERSION.get(record.levelno, LEVEL_CONVERSION[logging.INFO])
 
         filename = None
         line = None
         col = None
-        match = re.match(r"(.+):(\d+):(\d+)", getattr(record, "annotate", ""))
+
+        if getattr(record, "gh_annotate_skip", False): # disable annotations for any gm4 log events flagged manually
+            return f"{level.capitalize()}: {record.name} {expl}" # formatted to resemble annotated entry
```

Comment:
Could we simply skip all INFO logs from getting annotated?

Reply:
GitHub stops annotating after 10 warnings and 10 notices (info equivalent), so we can get *some* nicely printed info messages. I wanted to ensure the more useful ones (from mecha for example) weren't drowned out by the normal log messages.